recipes-devtools/meson/meson.inc

```diff
@@ -0,0 +1,33 @@
+HOMEPAGE = "http://mesonbuild.com"
```

Comment:
Can we use meson from external meta-layers instead of copying the code here?

Reply:
It is from poky layer. current version in poky/meta is meson_0.53.2, hence added this